charts/reform-scan-blob-router/values.yaml

```diff
@@ -41,8 +41,8 @@ java:
 
     BULK_SCAN_PROCESSOR_URL: http://bulk-scan-processor-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
 
-    SMTP_HOST: "false"
-    SEND_DAILY_REPORT_ENABLED: "false"
+    SEND_DAILY_REPORT_ENABLED: "true"
```

Comment:
if prod only then need to go into flux config instead

Reply:
and SMTP_HOST - also to flux?

Reply:
microsoft provides that value openly so it is a nobrainer to google the value

Reply:
I will close this PR